apis/client/v1beta1/openstackclient_types.go

```diff
@@ -56,6 +57,13 @@ type OpenStackClientSpecCore struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// Secret containing any CA certificates which should be added to deployment pods
 	tls.Ca `json:",inline"`
+
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +listType=map
+	// +listMapKey=name
+	// List of environment variables to set in the container.
+	Env []corev1.EnvVar `json:"env,omitempty"`
```

Comment:
I see some APIs also use kubebuilder annotations for patchMergeKey and patchStrategy. So perhaps something like: Env []corev1.EnvVar `json:"env,omitempty" patchMergeKey:"name" patchStrategy:"merge"`

Reply:
done